pkg/controller/import-controller.go

```diff
@@ -417,7 +417,7 @@ func (r *ImportReconciler) updatePvcFromPod(pvc *corev1.PersistentVolumeClaim, p
 		pvc.GetLabels()[common.CDILabelKey] = common.CDILabelValue
 	}
 	if cc.IsPVCComplete(pvc) {
-		setLabelsFromTerminationMessage(pvc.GetLabels(), termMsg)
+		pvc.SetLabels(setLabelsFromTerminationMessage(pvc.GetLabels(), termMsg))
```

Comment:
Nit: `setLabelsFromTerminationMessage` sounds slightly innaccurate since it's no longer setting the labels, but returning them instead. This is personal preferrence though, just a thought.

Reply:
Do you think `addLabelsFromTerminationMessage` sound more accurate?

Reply:
Renamed it and updated the PR.

Reply:
Nice, thank you!